.github/workflows/release.yml

```diff
@@ -24,20 +25,23 @@ jobs:
       - name: Set up Docker Buildk
         uses: docker/setup-buildx-action@v1
 
-      - name: Login to Registry
+      - name: Login to ghcr
         uses: docker/login-action@v1
         with:
-          registry: registry.scality.com
-          username: ${{ secrets.REGISTRY_LOGIN }}
-          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
           file: ./Dockerfile
           push: true
-          tags: "registry.scality.com/artifacts/artifacts:${{ github.event.inputs.tag }}"
+          build-args: |
+            VERSION=${{ inputs.tag }}
```

Comment:
Why did you add this build argument? Just noticed it wasn't set before.

Reply:
I saw it in another project that was doing the same thing, and I found it interesting to copy it as well. Do you want me to remove it?

Reply:
It's only useful if a proper `ARG VERSION` is defined in the `Dockerfile` like:

```dockerfile
ARG VERSION
```
And later on used by the service itself.

In this is case it is not set and even if set, it will not be used by the service so I see no point in keeping it:
```suggestion
```